lnrpc/rpc.proto

```diff
@@ -1911,17 +1980,56 @@ message FundingShimCancel {
     bytes pending_chan_id = 1;
 }
 
+message FundingPsbtVerify {
+    /**
+    The funded but not yet signed PSBT that sends the exact channel capacity
+    amount to the PK script returned in the open channel message in a previous
+    step.
+    */
+    bytes funded_psbt = 1;
+
+    /// The pending channel ID of the channel to get the PSBT for.
+    bytes pending_chan_id = 2;
+}
+
+message FundingPsbtFinalize {
+    /**
+    The funded PSBT that contains all witness data to send the exact channel
+    capacity amount to the PK script returned in the open channel message in a
+    previous step.
+    */
+    bytes signed_psbt = 1;
```

Comment:
To make the API harder to use in an invalid manner, what if we just accepted the signatures/witnesses here, and then execute the finalization steps ourselves internally? Otherwise, it feels like the user can just skip straight to this step (assuming it still does verification).